main/src/cgeo/geocaching/ui/ContinuousRangeSlider.java

```diff
@@ -64,6 +73,70 @@ private void init() {
         inflate(ctw, R.layout.continuousrangeslider_view, this);
         binding = ContinuousrangesliderViewBinding.bind(this);
         setScale(0f, 100f, null, 11);
+        binding.sliderInternal.addOnSliderTouchListener(new RangeSlider.OnSliderTouchListener() {
+            int lastThumb = -1;
+            Float lastValue = -1.0f;
+            Float grace = null;
+
+            @Override
+            public void onStartTrackingTouch(final @NonNull RangeSlider slider) {
+                if (grace == null) {
+                    grace = (slider.getValueTo() - slider.getValueFrom()) / 20.0f;
+                }
+                lastThumb = slider.getActiveThumbIndex(); // on touch start it's the active thumb
+                final List<Float> values = slider.getValues();
+                if (lastThumb >= 0 && lastThumb < values.size()) {
+                    lastValue = values.get(lastThumb);
+                } else {
+                    lastValue = null;
+                }
+            }
+
+            @Override
+            public void onStopTrackingTouch(final @NonNull RangeSlider slider) {
+                final List<Float> values = slider.getValues();
+                if (slider.getFocusedThumbIndex() == lastThumb) { // on touch end it's the focussed thumb, though!
+                    // still on the same thumb, and not having moved it out of grace area?
+                    if (lastThumb >= 0 && lastThumb < values.size() && Math.abs(lastValue - values.get(lastThumb)) < grace) {
+                        final String title = String.format(getContext().getString(R.string.number_input_title), String.format(Locale.getDefault(), "%.2f", slider.getValueFrom()), String.format(Locale.getDefault(), "%.2f", slider.getValueTo()));
```

Comment:
This should be sparable if you use the TextParam.id - variant in the SimpleDialog call later. This method allows you to pass a resource id (R.string.number_input_title) followed by parameter values.

Reply:
done